extra-python/yarl/spec

```diff
@@ -1,3 +1,4 @@
 VER=1.6.3
+REL=1
```

Comment:
Why?

Reply:
yarl depends on multidict.

Reply:
A direct dependency does not imply necessity to bump rel...

Reply:
Please revert this change. Blocking merging for now.